api/src/main/java/cz/fi/muni/pa165/dto/HunterDTO.java

```diff
@@ -0,0 +1,57 @@
+package cz.fi.muni.pa165.dto;
+
+import com.fasterxml.jackson.annotation.JsonView;
+import cz.fi.muni.pa165.enums.Rank;
+import cz.fi.muni.pa165.views.View;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Created by Erik Macej 433744 , on 19.11.16.
+ *
+ * @author Erik Macej 433744
+ */
+public class HunterDTO extends UserDTO {
```

Comment:
I am not sure if we want to register users through HunterDTO or to create DTO especially for registering...